app/errors.py

```diff
@@ -0,0 +1,18 @@
+class VaccineError(Exception):
+    def __init__(self, message: str = "Default VaccineError message") -> None:

```

Comment:
Use a more descriptive default error message. For instance, 'An error occurred with the vaccine.' instead of 'Default VaccineError message'.